packages/drivers/odsp-driver/src/opsCaching.ts

```diff
@@ -135,9 +135,9 @@ export class OpsCache {
                         break;
                     }
                     if (messages.length === 0) {
-                        if (op.sequenceNumber > from + 1) {
```

Comment:
Note: That's defiantly a bug - it treated "from" as exclusive boundary, while all the code has been switched to treat "from" as inclusive. Added asserts in other layers to catch issues like that, such that we can find them sooner (and validate this code change as well).